pkg/util/pki/csr.go

```diff
@@ -418,35 +418,25 @@ func SignCertificate(template *x509.Certificate, issuerCert *x509.Certificate, p
 // SignCSRTemplate signs a certificate template usually based upon a CSR. This
 // function expects all fields to be present in the certificate template,
 // including it's public key.
-// It returns the certificate data followed by the CA data, encoded in PEM format.
-func SignCSRTemplate(caCerts []*x509.Certificate, caKey crypto.Signer, template *x509.Certificate) ([]byte, []byte, error) {
+// It returns the PEM bundle containing certificate data and the CA data, encoded in PEM format.
```

Comment:
nitpick: I wasn't sure what "PEM bundle" meant ("PEM-encoded certificate bundle" is a thing, but I feel confused with just "PEM bundle" 😅)

```suggestion
// Returns the certificate bundle containing both the PEM-encoded certificate and the
// PEM-encoded CA.
```

Reply:
:sweat_smile: Not sure what else it could mean..

`PEMBundle` is the name of the struct which it returns, so it returns _the_ PEM bundle which contains the certificate data and CA data.

Reply:
"PEM bundle" was immediately clear to me personally, I'd be fine with the original text

Reply:
I withdraw my nitpick comment since I agree: "PEM bundle" unequivocally means "PEM-encoded certificate bundle" 👍